sbi/simulators/simutils.py

```diff
@@ -1,22 +1,25 @@
 from __future__ import annotations
 
-from typing import Callable, Optional
+from typing import Callable, Optional, Tuple
 
 import torch
 from torch import Tensor
 from tqdm.auto import tqdm
-from sbi.simulators.mp_simulator import simulate_mp
+from sbi.simulators.mp_simutils import simulate_mp
```

Comment:
Why not simutils, I am curious? (continuing with the argument above about not repeating code, it helps a little bit if the concerned functions live in the same module?)

Reply:
My rational was to have a dedicated function which is using the `multiprocessing` python module. So if we moved to some other library we need to touch only this module.

Reply:
dedicated function OK, but I don't see why we should give every function a module...

Reply:
Sorry, I had meant module in my previous comment.